src/custom/components/Header/MobileMenuIcon/index.tsx

```diff
@@ -0,0 +1,82 @@
+import styled, { css } from 'styled-components/macro'
+
+const Wrapper = styled.div<{ isMobileMenuOpen: boolean; height?: number; width?: number; lineSize?: number }>`
+  z-index: 102;
+  display: flex;
+  cursor: pointer;
+  margin: 0 6px 0 16px;
+  position: relative;
+  width: ${({ width }) => (width ? width + 'px' : '34px')};
+  height: ${({ height }) => (height ? height + 'px' : '18px')};
```

Comment:
In these kinds of props i prefer setting the default in the function param destructuring like:
```ts
width: ${({ width = 18 }) => `${width}px`};
```

It just looks cleaner and is less code

Reply:
Ah nice, of course this is way cleaner setting the default like that!